### DISCUSSION TITLE
articles/linux/nix

### DISCUSSION THREAD

**Discussion by giscus** [lxl66566/lxl66566.github.io#22] [General]
# articles/linux/nix

安装与配置（NixOS 篇） 早在去年我便说过我的下一个操作系统很有可能是 NixOS。202405 的 OS 课需要做 PPT 汇报，我的选题是包管理器杂谈，又吹了一波 nix，把我自己心吹得痒痒的。 在考试期间由于压抑的氛围和不情愿的学习，平常想做的事的欲望会被放大许多。但是我预料到 NixOS 的安装肯定会非常折磨（好预测！），所以只在 WSL ...

https://absx.pages.dev/articles/linux/nix.html

**Comment by Ic-arbon** [comment]
我正在使用捞爆了的home-manager，快来克隆我的配置肘击我

**Reply by lxl66566** [reply]
有点少，简单克一个 [astronvim.nix](https://github.com/Ic-arbon/dotfiles/blob/main/modules/astronvim.nix)

**Reply by Ic-arbon** [reply]
那个是裸配置，fetch的是默认用户配置，我就处理了下依赖。so，你还得写lua，luckily we have [astrocommunity](https://github.com/AstroNvim/astrocommunity/tree/main/lua/astrocommunity/pack)
